examples/with-tailwind/package.json

```diff
@@ -13,10 +13,10 @@
     "prettier": "^3.1.1",
     "prettier-plugin-tailwindcss": "^0.5.10",
     "@repo/typescript-config": "workspace:*",
-    "turbo": "^1.12.1"
+    "turbo": "^1.12.4"
```

Comment:
I would prefer we do this consistently across all examples rather than one at a time

Reply:
absolutely. I will change it back to 1.12.1